conda/recipes/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: nvjitlink
+  version: 0.1.0
+
+source:
+  #url: https://github.com/rapidsai/nvjitlink/archive/refs/tags/0.1.0.tar.gz
```

Comment:
Sounds like we can drop this?

```suggestion
```

Reply:
I had thought that what would happen is that once satisfied here, we'd cut a release of `nvjitlink`, move this recipe to conda forge and uncomment this line pointing it to the release we want. Let me know if this sounds right.

Reply:
Yeah that's ok

Still would drop this commented line from here